test/batch-transaction.ts

```diff
@@ -181,7 +190,7 @@ describe('BatchTransaction', () => {
     const RESPONSE = {partitions: PARTITIONS};
 
     const QUERY = {a: 'b'};
-    const CONFIG = {reqOpts: QUERY};
+    const CONFIG = {reqOpts: QUERY, gaxOpts: {timeout: 1000}};
```

Comment:
```suggestion
    const CONFIG = {reqOpts: QUERY};
```

It doesn't look like this function is doing anything with `gaxOpts`, or that we're testing it anywhere, so having this defined here could be confusing.